docs/developers/how-can-i-restore-my-account.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How can I restore my Account?
+description: Guide to restoring an expired account
+lang: en
+---
+
+To restore your account which was swept away to the nether layer, you need to send a restore data to the restore server. 
```

Comment:
We updated the terminology from restore server to restoration node

Reply:
There are also many "restore server" below.

Reply:
I will fix this ; Btw I think this documentation uses 'restoration client'. Is this intentional?